modules/data-entry/src/main/frontend/src/questionnaireEditor/AnswerOptions.jsx

```diff
@@ -98,15 +98,24 @@ let AnswerOptions = (props) => {
       });
     }
 
-    setTempValue('');
+    tempValue && setTempValue('');
+
+    // Have to manually invoke submit with timeout to let re-rendering of adding new answer option complete
+    // Cause: Calling onBlur and mutating state can cause onClick for form submit to not fire
+    // Issue details: https://github.com/facebook/react/issues/4210
+    if (event?.relatedTarget.type == "submit") {
```

Comment:
This occasionally causes a console error because relatedTarget can be undefined.
```suggestion
    if (event?.relatedTarget?.type == "submit") {
```